ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+```
+
+Allows an anchor to communicate basic info about what currencies their `DIRECT_PAYMENT_SERVER` supports receiving from partner anchors.
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `description` fields in the response should be in this language.
+
+#### Response
+
+The response should be a JSON object like:
+
+```json
+{
+   "receive":{
+      "USD":{
+         "enabled":true,
+         "fee_fixed":5,
+         "fee_percent":1,
+         "min_amount":0.1,
+         "max_amount":1000,
+         "fields":{
+            "sender":{
+               "first_name":"The sender's first name",
+               "last_name":"The sender's last name"
+            },
+            "receiver":{
+               "first_name":"The receiver's first name",
+               "last_name":"The receiver's last name",
+               "email_address":"The receiver's email address"
+            },
+            "transaction":{
+               "routing_number":{
+                  "description":"routing number of the destination bank account"
+               },
+               "account_number":{
+                  "description":"bank account number of the destination"
+               },
+               "type":{
+                  "description":"type of deposit to make",
+                  "choices":[
+                     "SEPA",
+                     "SWIFT"
+                  ]
+               }
+            }
+         }
+      }
+   }
+}
+```
+
+The JSON object contains an entry for each asset that the anchor supports for receiving and completing a direct payment.
+
+#### For each asset available for receiving, response contains:
+
+* `min_amount`: Optional minimum amount. No limit if not specified.
+* `max_amount`: Optional maximum amount. No limit if not specified.
+* `fee_fixed`: Optional fixed (flat) fee for deposit. In units of the received asset. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_percent`: Optional percentage fee for deposit. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
+* `fields` object as explained below.
+
+The `fields` object allows an anchor to describe fields that must be passed into `POST /send`. It should include any KYC fields required for the sender or receiver, account information, and anything else required by the receiving anchor in order to complete a transaction.  Fields are broken out by `sender`, `receiver`, and `transaction`.  `sender` and `receiver` contain KYC requests of values from SEP-9 while `transacton` contains transaction specific information requested.
+
+Each `fields` sub-object contains a key for each field name and an object with the following fields as the value:
+
+* `description`: description of field to show to user.
+* `choices`: list of possible values for the field.
```

Comment:
I removed the ability to mark fields as 'optional'.  Do you think optional fields are useful?

Reply:
I'd keep it if unless we're sure it wouldn't be useful.

Reply:
I'd keep it if unless we're sure it wouldn't be useful.